README.md

```diff
@@ -51,13 +55,13 @@ resource "logdna_view" "my_view" {
 }
 ```
 
-Using the logdna_view resource- a user can create a view with a provided name query, hosts, category, tags, email, pagerduty and webhook channels, delete a view with a given viewid or update a view's properties.
+Using the `logdna_view` resource, a user can create a View with provided `name`, `query`, `hosts`, `categories`, `tags`, `email_channel`, `pagerduty_channel`, and `webhook_channel`, delete a View with a given `viewid` or update a View's properties.
```

Comment:
Note to verify that View and Alert and PagerDuty have been capitalized correctly

Reply:
In general, if the word connotes a feature in our platform and is referencing that feature explicitly, the word is capitalized (e.g., View, Alerts). In this case, PagerDuty is lowercase because it is part of a key name, which are all lowercase in the example provided.